media-gfx/graphviz/patches/graphviz-2.49.1.patchset

```diff
@@ -0,0 +1,94 @@
+From 47d4dfc62ae6ab2355e9b1b48200abb5728e3e7a Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Wed, 29 Sep 2021 10:42:47 +0000
+Subject: set Haiku font directories
+
+
+diff --git a/configure.ac b/configure.ac
+index fe3f2be..c7f3d3f 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -144,6 +144,7 @@ UWIN=no
+ CYGWIN=no
+ MINGW32=no
+ DARWIN9=no
++HAIKU=no
+ case "${host_os}" in
+   *uwin* )
+ 	UWIN=yes
+@@ -185,6 +186,16 @@ case "${host_os}" in
+ 		;;
+         esac
+ 	;;
++	*haiku* )
++	HAIKU=yes
++	HAIKU_SYSTEM_FONTPATH=`finddir B_SYSTEM_FONTS_DIRECTORY`
++	HAIKU_COMMON_FONTPATH=`finddir B_COMMON_FONTS_DIRECTORY`
++	HAIKU_USER_FONTPATH=`finddir B_USER_FONTS_DIRECTORY`
```

Comment:
nowadays we have B_SYSTEM_FONTS_DIRECTORY B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY B_USER_FONTS_DIRECTORY  B_USER_NONPACKAGED_FONTS_DIRECTORY

Reply:
`finddir -l` still gives a line for `B_COMMON_FONTS_DIRECTORY`, but I could change those

Reply:
Something like:
```diff
+	HAIKU_SYSTEM_NONPACKAGED_FONTPATH=`finddir B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY `
```

Reply:
Just checked, both `B_COMMON_FONTS_DIRECTORY` and `B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY` result in the same path: `/boot/system/non-packaged/data/fonts`

Reply:
yes, but the "COMMON" constants should not be used anymore. And B_USER_NONPACKAGED_FONTS_DIRECTORY will still be missing.

Reply:
Working on it already ;)